code/game/objects/items/storage/backpack.dm

```diff
@@ -853,17 +853,22 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 					return
 	. = ..()
 
-/obj/item/storage/backpack/marine/engineerpack/afterattack(obj/O as obj, mob/user as mob, proximity)
-	if(!proximity) // this replaces and improves the get_dist(src,O) <= 1 checks used previously
+/obj/item/storage/backpack/marine/engineerpack/afterattack(obj/target, mob/user, proximity)
+	if(!proximity) // this replaces and improves the get_dist(src,localbackpack) <= 1 checks used previously
```

Comment:
Comment has the old var you were using, but not a big deal

Reply:
removed